src/main/java/org/terasology/las/bases/BaseProvider.java

```diff
@@ -53,11 +53,15 @@ public void process(GeneratingRegion region) {
         region.setRegionFacet(BaseFacet.class, facet);
     }
 
-    private Region3i CreateBaseRegionFromVector(Vector3i centerBasePosition) {
-        return Region3i.createFromMinMax(new Vector3i(centerBasePosition.x() - LASUtils.BASE_EXTENT, centerBasePosition.y(), centerBasePosition.z() - LASUtils.BASE_EXTENT), new Vector3i(centerBasePosition.x() + LASUtils.BASE_EXTENT, centerBasePosition.y(), centerBasePosition.z() + LASUtils.BASE_EXTENT));
+    private BlockRegion CreateBaseRegionFromVector(Vector3i centerBasePosition) {
+        return new BlockRegion(centerBasePosition.x() - LASUtils.BASE_EXTENT, centerBasePosition.y(),
+                centerBasePosition.z() - LASUtils.BASE_EXTENT, centerBasePosition.x() + LASUtils.BASE_EXTENT,
+                centerBasePosition.y(), centerBasePosition.z() + LASUtils.BASE_EXTENT);
     }
 
-    private Region3i CreateFlagRegionFromVector(Vector3i centerBasePosition) {
-        return Region3i.createFromMinMax(new Vector3i(centerBasePosition.x(), centerBasePosition.y() + 1, centerBasePosition.z()), new Vector3i(centerBasePosition.x(), centerBasePosition.y() + 1, centerBasePosition.z()));
+    private BlockRegion CreateFlagRegionFromVector(Vector3i centerBasePosition) {
+        return new BlockRegion(centerBasePosition.x(), centerBasePosition.y() + 1,
+                centerBasePosition.z(), centerBasePosition.x(), centerBasePosition.y() + 1,
+                centerBasePosition.z());
```

Comment:
Same here: at least rename the method, but I'm wondering whether we actually need it?

```suggestion
    private BlockRegion createFlagRegionFromVector(Vector3i centerBasePosition) {
        return new BlockRegion(centerBasePosition).translate(0, 1, 0);
```